### DISCUSSION TITLE
Why force a "sunset position"?

### DISCUSSION THREAD

**Discussion by Naturnobert** [basbruss/adaptive-cover#235] [Ideas]
Hey basbruss!

Happy to see that you're still tinkering on this whole idea.
I'm using your cover control now since you started with it. Today I checked the hassio thread again and found out that there is this HACS integration.

With your integration I'm now able to delete a lot of automations. Thank you very much!
But why do you force a "sunset position"? Is there a way to disable it?

Maybe the better question would be: How to Set "end time" for north West south and east based on sun azimuth Angle? (After sun is out of a windows fov)

Greetings,
naturnorbert

**Comment by basbruss** [comment]
Hi,

The sunset position is introduced to have two types of default since most people want their blinds to be in a different position when there is daylight than after sunset. You are still able to set the sunset default equal to the normal default to have essentially one default value.

> Maybe the better question would be: How to Set "end time" for north West south and east based on sun azimuth Angle? (After sun is out of a windows fov)

If you set both default options to the same value this should work instantly.
